src/quo/components/profile/expanded_collectible/style.cljs

```diff
@@ -7,10 +7,11 @@
    :border-radius   16})
 
 (defn image
-  [square? aspect-ratio]
-  {:width         "100%"
-   :aspect-ratio  (if square? 1 aspect-ratio)
-   :border-radius 16})
+  [square? aspect-ratio theme]
+  {:width            "100%"
+   :aspect-ratio     (if square? 1 aspect-ratio)
+   :border-radius    16
+   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)})
```

Comment:
Just adding the background color for collectibles

Reply:
[Figma design](https://www.figma.com/design/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?node-id=13212-177720&m=dev) also has a border

Reply:
Thank you @ajayesivan !

Actually it was a missing style in the component (`expanded-collectible`) not only for the transparent images, so I added it :+1: